spec/factories/ems_folder.rb

```diff
@@ -69,7 +69,7 @@
     hidden true
   end
 
-  factory :vmware_datacenter, :parent => :vmware_folder, :class => "Datacenter" do
+  factory :vmware_datacenter, :class => "Datacenter" do
```

Comment:
I don't like the removal of this.  A vmware_datacenter *is* a vmware_folder, and this is supposed to be used for inheriting the other attributes.  Why was this necessary to remove?